.github/workflows/extraction-eval.yml

```
@@ -0,0 +1,283 @@
+          release-tag: ${{ vars.BENCHMARK_BOT_RELEASE_TAG }}
+          require-subcommand: post-extraction-pr-comment
+        continue-on-error: true
+
+      - name: Post extraction-quality PR comment
+        if: always() && steps.bot.outcome == 'success' && github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
+        env:
+          GITHUB_TOKEN: ${{ secrets.BENCHMARK_BOT_APP_PRIVATE_KEY != '' && secrets.GITHUB_TOKEN || secrets.GITHUB_TOKEN }}
+          EXTRACTION_BENCHMARK_DATA_PATH: web/public/benchmarks/extraction-quality-benchmark-data.json
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          BOT_BIN: ${{ steps.bot.outputs.bot-bin }}
```

actionlint (1.7.12)

[error] property "bot-bin" is not defined in object type {bot-path: string; source: string}

(expression)